samples/udpsrv.lua

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env lua
+
+--
+-- Sample UDP server. Listens on 3333 port
+-- Use netcat to send strings to server (ncat -u 127.0.0.1 3333)
+--
+
+local lgi = require 'lgi'
+local GLib = lgi.GLib
+local Gio = lgi.Gio
+
+local app = Gio.Application{application_id = 'org.v1993.udptest',
+			    flags = Gio.ApplicationFlags.NON_UNIQUE}
+
+local socket = lgi.Gio.Socket.new(Gio.SocketFamily.IPV4,
+				  Gio.SocketType.DATAGRAM,
+				  Gio.SocketProtocol.UDP)
+local sa = lgi.Gio.InetSocketAddress.new(
+   Gio.InetAddress.new_loopback(Gio.SocketFamily.IPV4),
```

Comment:
Any reason why one of these uses "ipv4 loopback" and the other one uses "parse 127.0.0.1"? Is it just to show that both options exist or is there a difference between them?

Reply:
I'll just say there is no technical difference.

Reply:
@TingPing Do you think one of the two forms is better?

I think the "parse this IP address"-form is better for an example since it can easily be changed to other IPs. I'm not sure if that is a good argument or not.